datacollection/openapi/v3/schema/adsb.yaml

```diff
@@ -61,35 +56,307 @@ components:
+          magHeading:
+            type: integer
+            enum: [0,1]
+            description: True north or magnetic north
+              0 = True north
+              1 = Magnetic north
+          utcCoupledCondition:
+            type: integer
+            enum: [0,1]
+            description: Represents if the Ground Station is UTC-Coupled 
+              0 = Ground Station is not UTC coupled
+              1 = Ground Station is UTC coupled
+
+  #    ADSB1090ES:
+  #      type: object
+  #      properties:
+          surveilStatus:
+            type: integer
+            enum: [0,1,2,3]
+            description: Surveillance status
+              0 = No Condition
+              1 = permanent alert
+              2 = temp alert
+              3 = SPI
+          baroaltDiffMM:
+            type: integer
+            description: Difference between the pressure altitude and the gnss altitude in mm
+# not a java valid name. Can't ingest it right now. 2/24/2020
+#          1090esMopsVersion:
+#            type: integer
+#            enum: [1,2]
+#            description: 1 = DO-260A, 2 = DO-260B           
+  #    ADSBUAT:
+  #      type: object
+  #      properties:
+          secondaryAltType:
+            type: integer
+            enum: [0,1]
+            description: Altitude source
+              0 = Pressure
+              1 = Geometric
+          secondaryAltitudeMM:
+            type: integer
+            description: Geometric altitude or barometric pressure altitude in millimeters
+          tisBSiteId:
+            type: integer
+            description: The tisBSiteId is unit-less and is from the a transmitted TISb UAT message signifies which uplink tower transmitted the TISb frame
+          transmitMSO:
+            type: integer
+            enum: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63]
+            description: the transmitMSO is the 6bit field from the transmitted UAT message which should signify which MSO the message was transmitted in.  MSO's can range from 0 to 3951 but only transmit the 6 LSB's of the actual MSO if transmitted.
+          addressQualifier:
+            type: integer
+            enum: [0,1,2,3,4,5,6,7]
+            description: Defines the type of target that delivered the data
+              0 = ADS-B target with ICAO 24-bit
+              1 = Reserved for National use
+              2 = TIS-B target with ICAO 24-bit address
+              3 = TIS-B target with track file identifier
+              4 = Surface Vehicle
+              5 = Fixed ADS-B Beacon
+              6-7 = (Reserved)
+          uatMopsVersion:
+            type: integer
+            enum: [0,1]
+            description: 1 = DO-282A, 2 = DO-282B
+          callSignID:
+            type: integer
+            enum: [0,1]
+            description: 0 = Fightplan, 1 = CallSign
+
+  #    RID:
```

Comment:
what is this

Reply:
That looks like a typographical error: should be Flightplan. I'll fix it.